ERROR 2019-01-21 10:06:14 _log Exception: Error reading SSH protocol banner
ERROR 2019-01-21 10:06:14 _log Traceback (most recent call last):
ERROR 2019-01-21 10:06:14 _log   File "/app/.pyenv/versions/test-3.6/lib/python3.6/site-packages/paramiko/transport.py", line 2044, in _check_banner
ERROR 2019-01-21 10:06:14 _log     buf = self.packetizer.readline(timeout)
ERROR 2019-01-21 10:06:14 _log   File "/app/.pyenv/versions/test-3.6/lib/python3.6/site-packages/paramiko/packet.py", line 353, in readline
ERROR 2019-01-21 10:06:14 _log     buf += self._read_timeout(timeout)
ERROR 2019-01-21 10:06:14 _log   File "/app/.pyenv/versions/test-3.6/lib/python3.6/site-packages/paramiko/packet.py", line 555, in _read_timeout
ERROR 2019-01-21 10:06:14 _log     raise socket.timeout()
ERROR 2019-01-21 10:06:14 _log socket.timeout
ERROR 2019-01-21 10:06:14 _log 
ERROR 2019-01-21 10:06:14 _log During handling of the above exception, another exception occurred:
ERROR 2019-01-21 10:06:14 _log 
ERROR 2019-01-21 10:06:14 _log Traceback (most recent call last):
ERROR 2019-01-21 10:06:14 _log   File "/app/.pyenv/versions/test-3.6/lib/python3.6/site-packages/paramiko/transport.py", line 1893, in run
ERROR 2019-01-21 10:06:14 _log     self._check_banner()
ERROR 2019-01-21 10:06:14 _log   File "/app/.pyenv/versions/test-3.6/lib/python3.6/site-packages/paramiko/transport.py", line 2049, in _check_banner
ERROR 2019-01-21 10:06:14 _log     'Error reading SSH protocol banner' + str(e)
ERROR 2019-01-21 10:06:14 _log paramiko.ssh_exception.SSHException: Error reading SSH protocol banner
ERROR 2019-01-21 10:06:14 _log 
ERROR 2019-01-21 10:06:14 connect 用户Administrator通过webssh连接10.1.19.12失败！原因：Error reading SSH protocol banner
ERROR 2019-01-21 10:06:48 _log Exception: Error reading SSH protocol banner
ERROR 2019-01-21 10:06:48 _log Traceback (most recent call last):
ERROR 2019-01-21 10:06:48 _log   File "/app/.pyenv/versions/test-3.6/lib/python3.6/site-packages/paramiko/transport.py", line 2044, in _check_banner
ERROR 2019-01-21 10:06:48 _log     buf = self.packetizer.readline(timeout)
ERROR 2019-01-21 10:06:48 _log   File "/app/.pyenv/versions/test-3.6/lib/python3.6/site-packages/paramiko/packet.py", line 353, in readline
ERROR 2019-01-21 10:06:48 _log     buf += self._read_timeout(timeout)
ERROR 2019-01-21 10:06:48 _log   File "/app/.pyenv/versions/test-3.6/lib/python3.6/site-packages/paramiko/packet.py", line 555, in _read_timeout
ERROR 2019-01-21 10:06:48 _log     raise socket.timeout()
ERROR 2019-01-21 10:06:48 _log socket.timeout
ERROR 2019-01-21 10:06:48 _log 
ERROR 2019-01-21 10:06:48 _log During handling of the above exception, another exception occurred:
ERROR 2019-01-21 10:06:48 _log 
ERROR 2019-01-21 10:06:48 _log Traceback (most recent call last):
ERROR 2019-01-21 10:06:48 _log   File "/app/.pyenv/versions/test-3.6/lib/python3.6/site-packages/paramiko/transport.py", line 1893, in run
ERROR 2019-01-21 10:06:48 _log     self._check_banner()
ERROR 2019-01-21 10:06:48 _log   File "/app/.pyenv/versions/test-3.6/lib/python3.6/site-packages/paramiko/transport.py", line 2049, in _check_banner
ERROR 2019-01-21 10:06:48 _log     'Error reading SSH protocol banner' + str(e)
ERROR 2019-01-21 10:06:48 _log paramiko.ssh_exception.SSHException: Error reading SSH protocol banner
ERROR 2019-01-21 10:06:48 _log 
ERROR 2019-01-21 10:06:48 connect 用户Administrator通过webssh连接10.1.19.12失败！原因：Error reading SSH protocol banner
INFO 2019-01-21 10:37:40 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 10:37:40 _log Authentication (publickey) successful!
INFO 2019-01-21 10:38:16 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 10:38:16 _log Authentication (publickey) successful!
INFO 2019-01-21 10:38:18 _log [chan 2] Opened sftp connection (server version 3)
INFO 2019-01-21 10:38:20 _log [chan 2] sftp session closed.
INFO 2019-01-21 10:38:44 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 10:38:44 _log Authentication (publickey) successful!
INFO 2019-01-21 10:38:46 _log [chan 2] Opened sftp connection (server version 3)
INFO 2019-01-21 10:38:48 _log [chan 2] sftp session closed.
INFO 2019-01-21 16:10:14 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 16:10:14 _log Authentication (publickey) failed.
INFO 2019-01-21 16:10:14 _log Authentication (password) successful!
INFO 2019-01-21 16:12:05 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 16:12:06 _log Authentication (publickey) failed.
INFO 2019-01-21 16:12:06 _log Authentication (password) successful!
INFO 2019-01-21 16:14:08 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 16:14:08 _log Authentication (publickey) failed.
INFO 2019-01-21 16:14:08 _log Authentication (password) successful!
INFO 2019-01-21 16:14:17 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 16:14:17 _log Authentication (publickey) failed.
INFO 2019-01-21 16:14:17 _log Authentication (password) successful!
INFO 2019-01-21 16:15:28 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 16:15:28 _log Authentication (publickey) failed.
INFO 2019-01-21 16:15:28 _log Authentication (password) successful!
ERROR 2019-01-21 16:22:28 run 连接超时！
ERROR 2019-01-21 16:22:42 connect 用户zz通过webssh连接10.1.19.113失败！原因：[Errno None] Unable to connect to port 22 on 10.1.19.113
ERROR 2019-01-21 16:25:20 connect 用户zz通过webssh连接10.1.19.113失败！原因：[Errno None] Unable to connect to port 22 on 10.1.19.113
ERROR 2019-01-21 16:30:31 connect 用户zz通过webssh连接10.1.19.113失败！原因：[Errno None] Unable to connect to port 22 on 10.1.19.113
INFO 2019-01-21 16:35:40 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 16:35:40 _log Authentication (publickey) failed.
INFO 2019-01-21 16:35:40 _log Authentication (password) successful!
INFO 2019-01-21 16:45:33 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 16:45:33 _log Authentication (publickey) successful!
INFO 2019-01-21 17:21:16 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 17:21:16 _log Authentication (publickey) failed.
INFO 2019-01-21 17:21:16 _log Authentication (password) successful!
INFO 2019-01-21 17:21:18 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 17:21:18 _log Authentication (publickey) failed.
INFO 2019-01-21 17:21:18 _log Authentication (password) successful!
INFO 2019-01-21 17:52:54 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-21 17:52:54 _log Authentication (publickey) failed.
INFO 2019-01-21 17:52:54 _log Authentication (password) successful!
INFO 2019-01-22 09:11:42 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-22 09:11:42 _log Authentication (publickey) failed.
INFO 2019-01-22 09:11:42 _log Authentication (password) successful!
INFO 2019-01-22 09:12:24 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-22 09:12:24 _log Authentication (publickey) failed.
INFO 2019-01-22 09:12:24 _log Authentication (password) successful!
INFO 2019-01-22 09:20:12 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-22 09:20:12 _log Authentication (publickey) failed.
INFO 2019-01-22 09:20:12 _log Authentication (password) successful!
INFO 2019-01-22 09:21:51 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-22 09:21:52 _log Authentication (publickey) failed.
INFO 2019-01-22 09:21:52 _log Authentication (password) successful!
INFO 2019-01-22 09:39:25 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-22 09:39:25 _log Authentication (publickey) failed.
INFO 2019-01-22 09:39:25 _log Authentication (password) successful!
ERROR 2019-01-22 16:51:04 run 连接超时！
INFO 2019-01-23 09:59:12 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-23 09:59:12 _log Authentication (publickey) failed.
INFO 2019-01-23 09:59:12 _log Authentication (password) successful!
INFO 2019-01-23 10:37:20 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-23 10:37:20 _log Authentication (publickey) successful!
ERROR 2019-01-23 11:02:10 record 数据库添加用户操作记录失败，原因：'record_file' is an invalid keyword argument for this function
INFO 2019-01-23 11:25:20 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-23 11:25:20 _log Authentication (publickey) successful!
ERROR 2019-01-23 15:57:16 run 连接超时！
ERROR 2019-01-23 15:58:50 run 连接超时！
ERROR 2019-01-23 16:00:45 run 连接超时！
ERROR 2019-01-23 16:02:18 run 连接超时！
ERROR 2019-01-23 16:13:43 run 连接超时！
INFO 2019-01-23 17:27:58 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-23 17:27:58 _log Authentication (publickey) failed.
INFO 2019-01-23 17:27:58 _log Authentication (password) successful!
INFO 2019-01-23 17:31:20 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-23 17:31:20 _log Authentication (publickey) failed.
INFO 2019-01-23 17:31:20 _log Authentication (password) successful!
INFO 2019-01-23 17:37:59 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-23 17:38:00 _log Authentication (publickey) failed.
INFO 2019-01-23 17:38:00 _log Authentication (password) successful!
INFO 2019-01-23 17:41:13 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-23 17:41:13 _log Authentication (publickey) failed.
INFO 2019-01-23 17:41:13 _log Authentication (password) successful!
INFO 2019-01-23 17:58:24 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-23 17:58:24 _log Authentication (publickey) failed.
INFO 2019-01-23 17:58:24 _log Authentication (password) successful!
INFO 2019-01-23 18:03:30 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-23 18:03:30 _log Authentication (publickey) failed.
INFO 2019-01-23 18:03:30 _log Authentication (password) successful!
INFO 2019-01-23 18:06:20 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-23 18:06:21 _log Authentication (publickey) failed.
INFO 2019-01-23 18:06:21 _log Authentication (password) successful!
INFO 2019-01-23 18:09:28 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-23 18:09:28 _log Authentication (publickey) failed.
INFO 2019-01-23 18:09:28 _log Authentication (password) successful!
ERROR 2019-01-24 09:14:08 connect 用户zz通过webssh连接10.1.19.113失败！原因：[Errno None] Unable to connect to port 22 on 10.1.19.113
INFO 2019-01-24 09:14:56 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-24 09:14:56 _log Authentication (publickey) failed.
INFO 2019-01-24 09:14:56 _log Authentication (password) successful!
ERROR 2019-01-24 09:21:04 connect 用户root通过webssh连接172.16.1.100失败！原因：[Errno 110] Connection timed out
ERROR 2019-01-24 09:23:51 connect 用户root通过webssh连接172.16.1.100失败！原因：[Errno 110] Connection timed out
ERROR 2019-01-24 09:24:40 connect 用户root通过webssh连接172.16.1.100失败！原因：timed out
ERROR 2019-01-24 09:25:00 connect 用户root通过webssh连接10.1.19.113失败！原因：[Errno None] Unable to connect to port 22 on 10.1.19.113
ERROR 2019-01-24 11:01:57 run 连接超时！
ERROR 2019-01-24 12:13:11 connect 用户zz通过webssh连接10.1.19.113失败！原因：[Errno None] Unable to connect to port 22 on 10.1.19.113
INFO 2019-01-24 14:35:09 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-24 14:35:09 _log Authentication (publickey) failed.
INFO 2019-01-24 14:35:09 _log Authentication (password) successful!
INFO 2019-01-24 14:52:14 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-24 14:52:14 _log Authentication (publickey) successful!
INFO 2019-01-24 15:59:43 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-24 15:59:43 _log Authentication (publickey) successful!
INFO 2019-01-24 16:04:25 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-24 16:04:25 _log Authentication (publickey) successful!
INFO 2019-01-24 16:06:01 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-24 16:06:01 _log Authentication (publickey) successful!
INFO 2019-01-24 16:09:05 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-24 16:09:05 _log Authentication (publickey) successful!
INFO 2019-01-24 16:16:17 _log Connected (version 2.0, client OpenSSH_5.3)
INFO 2019-01-24 16:16:17 _log Authentication (publickey) successful!
